python/rmm/tests/test_rmm.py

```diff
@@ -586,6 +586,34 @@ def test_cuda_async_memory_resource_threshold(nelem, alloc):
     array_tester("u1", 2 * nelem, alloc)  # should trigger release
 
 
+@pytest.mark.parametrize(
+    "mr", [rmm.mr.CudaMemoryResource, rmm.mr.CudaAsyncMemoryResource]
+)
+def test_limiting_resource_adaptor(mr):
+    if not _CUDAMALLOC_ASYNC_SUPPORTED:
+        pytest.skip("cudaMallocAsync not supported")
```

Comment:
Move this to the parameter:
```
pytest.param(rmm.mr.CudaAsyncMemoryResource, marks=pytest.mark.skipif(not _CUDAMALLOC_ASYNC_SUPPORTED, reason="cudaMallocAsync not supported"))
```

Reply:
I didn't remember this was possible, but it's indeed better. Done in https://github.com/rapidsai/rmm/pull/1327/commits/23921b16cca502ae1046ac69faf027cafd77111b .